MREUnityRuntime/MREUnityRuntimeLib/Core/ActorManager.cs

```diff
@@ -188,7 +199,8 @@ internal bool OnActorDestroy(Guid id)
 
 				actor.RigidBodyAdded -= OnRigidBodyAdded;
 				actor.RigidBodyRemoved -= OnRigidBodyRemoved;
-				actor.RigidBodyKinematicsChanged -= OnActorRigidBodyKinematicsChanged;
+				actor.RigidBodyKinematicsChanged -= OnRigidBodyKinematicsChanged;
+				actor.RigidBodyOwnerChanged += OnRigidBodyOwnerChanged;
```

Comment:
> += [](start = 31, length = 3)

I believe this should be the -= to unsubscribe like the others?

Reply:
absolutely. thanks